experimental/PropertyDDS/packages/property-shared-tree-interop/src/test/schemaConverter.spec.ts

```diff
@@ -0,0 +1,75 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+	brand,
+	FieldKinds,
+	createSchemaRepository,
+	fieldSchema,
+	lookupGlobalFieldSchema,
+	lookupTreeSchema,
+	ValueSchema,
+	rootFieldKey,
+} from "@fluid-internal/tree";
+import { PropertyFactory } from "@fluid-experimental/property-properties";
+import { convertPSetSchemaToSharedTreeLls } from "../schemaConverter";
+import personSchema from "./personSchema";
+
+describe("schema converter", () => {
+	beforeAll(() => {
+		PropertyFactory.register(Object.values(personSchema));
+	});
+
+	it(`can use "NodeProperty" as root`, () => {
+		const rootFieldSchema = fieldSchema(FieldKinds.optional, [
+			brand("NodeProperty"),
+			brand("NamedNodeProperty"),
+			brand("RelationshipProperty"),
+			brand("Test:Address-1.0.0"),
+			brand("Test:Person-1.0.0"),
+		]);
+		const schemaRepository = createSchemaRepository();
+		convertPSetSchemaToSharedTreeLls(schemaRepository, rootFieldSchema);
+
+		// 12 basic types + NodeProperty
+		expect(schemaRepository.globalFieldSchema.size).toEqual(1);
+		expect(lookupGlobalFieldSchema(schemaRepository, rootFieldKey)).toEqual(rootFieldSchema);
+
+		expect(schemaRepository.treeSchema.size).toEqual(31);
```

Comment:
The comment above states, you have 12+1 types but here you check for 31.

Reply:
Comment fixed.